server/blob/blob-union/src/test/java/org/apache/james/blob/union/UnionBlobStoreTest.java

```diff
@@ -81,6 +81,11 @@ public InputStream read(BucketName bucketName, BlobId blobId) {
             throw new RuntimeException("broken everywhere");
         }
 
```

Comment:
No tests for unionBlobStore::deleteBucket?

I would like at the very least one test:

```
Given a legacyBlobStore with 1 blob blob1 in bucket b1
And currentBlobStore with 1 blob blob2 in bucket b1
When b1 is deleted
Then blob1 & blob2 of b1 bucket cannot be read
```